client/package.json

```diff
@@ -19,7 +19,6 @@
     "test": "jest --config ./jest.config.js --maxWorkers=50% --env=jsdom",
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
-    "//": "The sed command here is a temporary workaround to fix an issue in graphql-codegen",
```

Comment:
offroad: should have been removed when the script was updated. possibly a merge issue.